pkg/packagekit/updates.jsx

```diff
@@ -504,9 +524,9 @@ class ApplyUpdates extends React.Component {
 
 const AskRestart = ({ onIgnore, onRestart, history }) => <>
     <EmptyStatePanel icon={RebootingIcon}
-                     title={ _("Restart Recommended") }
-                     paragraph={ _("Updated packages may require a restart to take effect.") }
-                     action={ _("Restart Now") }
```

Comment:
We need to keep this -- we can only tell *if* a reboot is needed, but we can't say for sure that it's *not needed*. We really need tracer and such for that. So let's keep this as a fallback for now.